Menus/BuyMenuGUI.cpp

```diff
@@ -1319,45 +1335,58 @@ void BuyMenuGUI::Update()
         }
 
         int listSize = m_pCartList->GetItemList()->size();
-        if (pressDown)
-        {
+        if (pressDown) {
             m_ListItemIndex++;
-            if (m_ListItemIndex >= listSize)
-            {
+            if (m_ListItemIndex >= listSize) {
                 m_ListItemIndex = listSize - 1;
                 // If at the end of the list and the player presses down, then switch focus to the BUY button
                 m_FocusChange = 1;
                 m_MenuFocus = OK;
-            }
-            // Only do list change logic if we actually did change
-            else
-            {
+            } else {
+                // Only do list change logic if we actually did change
                 m_pCartList->SetSelectedIndex(m_ListItemIndex);
                 g_GUISound.SelectionChangeSound()->Play(m_pController->GetPlayer());
             }
-        }
-        else if (pressUp)
-        {
+        } else if (pressUp) {
             m_ListItemIndex--;
-            if (m_ListItemIndex < 0)
-            {
+            if (m_ListItemIndex < 0) {
                 m_ListItemIndex = 0;
                 g_GUISound.UserErrorSound()->Play(m_pController->GetPlayer());
-            }
-            // Only do list change logic if we actually did change
-            else
-            {
+            } else {
+                // Only do list change logic if we actually did change
                 m_pCartList->SetSelectedIndex(m_ListItemIndex);
                 g_GUISound.SelectionChangeSound()->Play(m_pController->GetPlayer());
             }
         }
 
         // Get handle to the currently selected item, if any
         GUIListPanel::Item *pItem = m_pCartList->GetItem(m_ListItemIndex);
+        std::string description = "";
 
-        // Show popup info box next to selected item, but only if it has a description
-        if (pItem && pItem->m_pEntity && !pItem->m_pEntity->GetDescription().empty())
-        {
+        if (pItem && pItem->m_pEntity) {
+            //description = (pItem->m_pEntity->GetDescription() != "") ? pItem->m_pEntity->GetDescription() : "-No Information Found-";
```

Comment:
Seems like some junk code here.

Reply:
Yep, those comments were just test stuff, removed now. Screenshots above should give an example of it.